spec/specification/json_spec.rb

```diff
@@ -80,6 +81,15 @@ module Pod
 
         new_json.should.equal json
       end
+
+      it 'maintains correct order of keys across versions' do
```

Comment:
@segiddins this test fails when I add `hash.delete('platforms')` either inside or outside the conditional.